packages/wrangler/src/utils/worker-not-found-error.ts

```diff
@@ -0,0 +1,31 @@
+import { APIError } from "@cloudflare/workers-utils";
+
+/**
+ This is the error code from the Cloudflare API signaling that a worker could not be found on the target account
+ */
+export const WORKER_NOT_FOUND_ERR_CODE = 10007 as const;
+
+/**
+ This is the error code from the Cloudflare API signaling that a worker environment (legacy) could not be found on the target account
+ */
+export const WORKER_LEGACY_ENVIRONMENT_NOT_FOUND_ERR_CODE = 10090 as const;
```

Comment:
I added this for completeness / just-in-case, I don't think it's changing anything here, but I am happy to add a changeset for this if we want

Reply:
ah actually this turns out to help us handling both 10007 and 10090 errors consistently now 😄